src/Compilers/Core/Portable/Syntax/SyntaxNodeExtensions.cs

```diff
@@ -277,6 +277,7 @@ public static TRoot ReplaceTrivia<TRoot>(this TRoot root, SyntaxTrivia trivia, S
         }
 
         internal const string DefaultIndentation = "    ";
+        internal const string DefaultEOL = "\r\n";
```

Comment:
That's the way the current implementation is.   If we want to change the behavior we can think about that separately.